tests/UnifiedSpecTests/ManagesFailPointsTrait.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace MongoDB\Tests\UnifiedSpecTests;
+
+use MongoDB\Driver\Server;
+use MongoDB\Operation\DatabaseCommand;
+use stdClass;
+
+use function PHPUnit\Framework\assertIsString;
+use function PHPUnit\Framework\assertObjectHasAttribute;
+
+trait ManagesFailPointsTrait
+{
+    /** @var list<list{string, Server}> */
+    private array $failPointsAndServers = [];
+
+    public function configureFailPoint(stdClass $failPoint, Server $server): void
+    {
+        assertObjectHasAttribute('configureFailPoint', $failPoint);
+        assertIsString($failPoint->configureFailPoint);
+        assertObjectHasAttribute('mode', $failPoint);
+
+        $operation = new DatabaseCommand('admin', $failPoint);
+        $operation->execute($server);
+
+        if ($failPoint->mode !== 'off') {
```

Comment:
I don't believe this is used in any tests, but it seemed reasonable to check for it since there's no point in disabling a fail point twice.